doc/source/user/basics.creation.rst

```diff
@@ -9,55 +9,100 @@ Array creation
 
-There are 5 general mechanisms for creating arrays:
+There are 6 general mechanisms for creating arrays:
 
-1) Conversion from other Python structures (e.g., lists, tuples)
-2) Intrinsic numpy array creation objects (e.g., arange, ones, zeros,
+1) Conversion from other Python structures (i.e. lists and tuples)
+2) Intrinsic NumPy array creation functions (e.g. arange, ones, zeros,
    etc.)
-3) Reading arrays from disk, either from standard or custom formats
-4) Creating arrays from raw bytes through the use of strings or buffers
-5) Use of special library functions (e.g., random)
+3) Replicating, joining, or mutating existing arrays
+4) Reading arrays from disk, either from standard or custom formats
+5) Creating arrays from raw bytes through the use of strings or buffers
+6) Use of special library functions (e.g., random)
 
-This section will not cover means of replicating, joining, or otherwise
-expanding or mutating existing arrays. Nor will it cover creating object
-arrays or structured arrays. Both of those are covered in their own sections.
+You can use these methods to create  ndarrays or :ref:`structured_arrays`.
+This document will cover general methods for ndarray creation. 
 
-Converting Python array_like Objects to NumPy Arrays
-====================================================
-
-In general, numerical data arranged in an array-like structure in Python can
-be converted to arrays through the use of the array() function. The most
-obvious examples are lists and tuples. See the documentation for array() for
-details for its use. Some objects may support the array-protocol and allow
-conversion to arrays this way. A simple way to find out if the object can be
-converted to a numpy array using array() is simply to try it interactively and
-see if it works! (The Python Way).
-
-Examples: ::
-
- >>> x = np.array([2,3,1,0])
- >>> x = np.array([2, 3, 1, 0])
- >>> x = np.array([[1,2.0],[0,0],(1+1j,3.)]) # note mix of tuple and lists,
-     and types
- >>> x = np.array([[ 1.+0.j, 2.+0.j], [ 0.+0.j, 0.+0.j], [ 1.+1.j, 3.+0.j]])
+1) Converting Python sequences to NumPy Arrays
+===============================================
 
-Intrinsic NumPy Array Creation
-==============================
+NumPy arrays can be defined using Python sequences such as lists and
+tuples. Lists and tuples are defined using ``[...]`` and ``(...)``,
+respectively. Lists and tuples can define ndarray creation::
+* a list of numbers will create a 1D array, 
+* a list of lists will create a 2D array, 
+* further nested lists will create higher-dimensional arrays. In general,
+  any array object is called an **ndarray** in NumPy.
 
-NumPy has built-in functions for creating arrays from scratch:
+::
 
-zeros(shape) will create an array filled with 0 values with the specified
-shape. The default dtype is float64. ::
+ >>> a1D = np.array([1, 2, 3, 4])
+ >>> a2D = np.array([[1, 2], [3, 4]])
+ >>> a3D = np.array([[[1, 2], [3, 4]],
+                     [[5, 6], [7, 8]]])
```

Comment:
Did you mean to indent this block?

Reply:
> Did you mean to indent this block?

Yes, I was following the convention I found in the `basics.creation.rst` where the code was indented with a space in the examples:

```
::

\s>>> example = 1
```

Should they have a 4-space indent for PEP-8?

Reply:
That would be nicer, I thought the failure to build the docs that pointed to line 34 was because there is no indentation. I missed the single space :(

Reply:
> That would be nicer, I thought the failure to build the docs that pointed to line 34 was because there is no indentation. I missed the single space :(

I missed it at first too, looking at `absolute_beginners.rst` the code is indented with two space-indents `\s\s`. 

Q: Should there be a Sphinx indentation standard in the docs?